src/shaders/hillshade.fragment.glsl

```diff
@@ -26,10 +27,23 @@ void main() {
     float azimuth = u_light.y + PI;
     float polar = u_light.z;
 
+    // We scale the slope exponentially based on intensity, using a calculation similar to 
+    // the exponential interpolation function in the style spec: 
+    // https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/expression/definitions/interpolate.js#L217-L228
+    // so that higher intensity values create more opaque hillshading.
+    slope = intensity != 0.5 ? 
+        ((pow(1.875 - intensity * 1.75, slope) - 1.0) / (pow(1.875 - intensity * 1.75, (0.5 * PI)) - 1.0)) * (0.5 * PI) 
```

Comment:
I think it'd be helpful to have more variables here to make this more readable( i.e `float base = 1.875 - intensity * 1.75;`)